networkx/algorithms/isomorphism/isomorph.py

```diff
@@ -33,13 +33,15 @@ def could_be_isomorphic(G1, G2):
     # Check local properties
     d1 = G1.degree()
     t1 = nx.triangles(G1)
-    c1 = nx.number_of_cliques(G1)
+    clqs_1 = list(nx.find_cliques(G1))
+    c1 = {n: sum(1 for c in clqs_1 if n in c) for n in G1}  # number of cliques
     props1 = [[d, t1[v], c1[v]] for v, d in d1]
     props1.sort()
 
     d2 = G2.degree()
     t2 = nx.triangles(G2)
-    c2 = nx.number_of_cliques(G2)
+    clqs_2 = list(nx.find_cliques(G2))
+    c2 = {n: sum(1 for c in clqs_2 if n in c) for n in G2}  # number of cliques
```

Comment:
I guess we could save some memory by avoiding the creation of the list. But the current `number_of_cliques` doesn't do it, so we could leave it this way too.
```suggestion
    # number of cliques for each node
    c2 = {n: sum(1 for c in nx.find_cliques(G2) if n in c) for n in G2}
```

Reply:
Sorry -- my suggestion of inlining `find_cliques` into the dict comprehension is a bad suggestion. That would call the same (long) function repeatedly for each node instead of once at the beginning.  I'm going to merge this based on @paulitapb and my approval.